api/lua/pinnacle/grpc/defs.lua

```diff
@@ -1360,6 +1360,10 @@ local pinnacle_v1_Backend = {
 ---@field tag_id integer?
 ---@field set_or_toggle pinnacle.util.v1.SetOrToggle?
 
+---@class pinnacle.tag.v1.SwitchOutputRequest
+---@field output_name string?
+---@field tag_names string[]?
```

Comment:
This should be the tag ids to match protobuf definition, not their names'
You should not edit this file, but run
`just build` or `just gen-lua-pb-defs` to generate it